code/modules/mob/mob.dm

```diff
@@ -615,6 +615,8 @@
 					continue
 				if(overrides.len && (A in overrides))
 					continue
```

Comment:
While you're in here you could make this apply the override image as an atom rather then hide the atom

Right now you can still meta blood as an ai by the fact the blood does not show up at all in this list. 

But it might require a seperate PR because you would also have to go thru every override image and make sure it changes it's name to match (as well as fun shit like match up directions)

Reply:
sounds unfun